src/components/Resume/ResumeCreationApi.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export async function PostResume(resume) {
```

Comment:
Only fuctional components start with uppercase, **postResume** is the correct name